src/Screens/EditChoreScreen.tsx

```diff
@@ -1,19 +1,24 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
 import * as React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Text, TextInput } from 'react-native-paper';
+import { RadioButton, Text, TextInput, Modal, Button, Portal } from 'react-native-paper';
 import BigButton from '../Components/Buttons/BigButton';
 import ChoreCard from '../Components/Cards/ChoreCard';
 import { getTheme } from '../Components/theme';
 import { ChoreCreate } from '../Data/chore';
 import { RootStackParamList } from '../Navigation/RootNavigator';
-import { editChore } from '../Store/choreSlice';
+import { editChore, setChoreFrequency } from '../Store/choreSlice';
```

Comment:
setChoreFrequency doesn't seem to be used anywhere, you can probably remove this import